### PR TITLE
Fix document renderer sizing issues (3/4)

### PR DESCRIPTION
**Pull request by mrdrogdrog**
### Component/Part
Markdown document renderer

### Description
This PR fixes the sizing issues in the document renderer. Documents should now have a minimal width if possible.

### Steps

- [x] Added implementation
- [x] I read the [contribution documentation](https://github.com/hedgedoc/react-client/blob/main/CONTRIBUTING.md) and signed-off my commits to accept the DCO.

### CODE REVIEW

**Review by DerMolly — COMMENTED** [review]
Seems fine so far.

Maybe you could explain a few things for me:

- why reduce the 1280 width to 1100?
- In the commit message you mention moving the yaml metadata error, but I can't seem to find that in this PR.

**Comment by mrdrogdrog** [comment]
> Seems fine so far.
> 
> Maybe you could explain a few things for me:
> 
> * why reduce the 1280 width to 1100?

900 (width of the md content) + 200 (min width of the sides) = 1100

> * In the commit message you mention moving the yaml metadata error, but I can't seem to find that in this PR.

Fixed that.

**Review by DerMolly — APPROVED** [review]

File: src/components/editor-page/editor-pane/one-dark.scss [hunk 1]
```diff
@@ -1,11 +1,7 @@
...
- * SPDX-License-Identifier: MIT
+/*!
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
  *
- * Atom One Dark Theme
- * GitHub https://github.com/hikio/brackets-one-dark
+ * SPDX-License-Identifier: AGPL-3.0-only
  */
 
 /*
```
Comment (ErikMichelson) [inline src/components/editor-page/editor-pane/one-dark.scss:7]: Wrong again as in clean-up-2 (https://github.com/hedgedoc/react-client/pull/987#discussion_r567330083)

**Review by ErikMichelson — CHANGES_REQUESTED** [review]
With exception of the copyright thing, this seems fine.

File: src/components/editor-page/editor-pane/one-dark.scss (hunk 1 above)
Comment (mrdrogdrog) [reply src/components/editor-page/editor-pane/one-dark.scss:7]: will automaticly be fixed when clean-up-2 is merged

Comment (mrdrogdrog) [reply src/components/editor-page/editor-pane/one-dark.scss:7]: Rebased

**Comment by ErikMichelson** [comment]
In certain splitter positions the shadow of the splitter overlays the text, making the rendered view looking really compressed. A bit more padding would be gerat.

image

**Review by ErikMichelson — CHANGES_REQUESTED** [review]
After dragging the splitter once, the movement handler seems to not get removed, resulting in move of the splitter every time you hove it.

**Comment by mrdrogdrog** [comment]
I reverted the changes that I made to the splitter and added another way to fix the space issue.
I guess the splitter could use some refactoring.

**Review by ErikMichelson — APPROVED** [review]

**Pull request merged**